src/bikesList.tsx

```diff
@@ -0,0 +1,222 @@
+    };
+    const deleteClicked = () => {
+      const newList = list.filter((item) => item.ID != list[selectedIndex].ID);  
+      setList(newList);
+      setOpenSlidingWindow(false);  
+    };
+    const handleClickOpen = () => {
+      setOpen(true);
+    };
+  
+    const handleClose = () => {
+      setOpen(false);
+    };
+    const handleAddBike = () => {
+      postBike({ID: Math.floor(Math.random() * 100), State: newBikeState, StationID: newBikeStationID}).then(r => {        
+    });
+      setOpen(false);
+    };
+    return (
+      <div style={{  height: "91vh", display: "flex", flexDirection: "column" }}>  
+        <List className={classes.ListSyle} subheader={<li/>}>
+        <li className={classes.listSection}>
+            <ul className={classes.ul}>
+              <ListSubheader style={{backgroundColor:'#4E4E50',display:'flex',fontWeight:'bold',height:'50px'}}>
+                <Box display="flex" flexDirection="row" p={1} m={1} alignSelf="center" style={{width:'90%'}}>
+                        <Box p={1} width="1%" >
+                          ID
+                        </Box>
+                        <Box p={1} width="10%">
+                          Bike State
+                        </Box>
+                        <Box p={1} width="7%">
+                          Station ID
+                        </Box>
+                      </Box>
+                      <Button startIcon={<AddIcon/>} style={{backgroundColor: 'white'}} onClick={handleClickOpen}>  
+                        Add new bike
+                      </Button>
+                      <Dialog disableBackdropClick  open={open} onClose={handleClose}>
+                        <DialogTitle>Fill the form</DialogTitle>
+                        <DialogContent>
+                        <form className={classes.container}>
+                          <FormControl className={classes.formControl}>   
+                            <InputLabel htmlFor="demo-dialog-native">
+                              State
+                            </InputLabel>
+                            <Select native value={newBikeState} onChange={handleChange} input={<Input />}>
+                              <option value={0}> Working </option>
+                              <option value={1}> In Service </option>
+                              <option value={2}> Blocked </option>
+                            </Select>
+                          </FormControl>
+                          <FormControl className={classes.formControl}>   
+                            <InputLabel htmlFor="demo-dialog-native">
+                              StationID
+                            </InputLabel>
+                            <Select native value={newBikeStationID} onChange={handleChangeStation} input={<Input />}>
+                              {stations.map((station)=>{
+                                return (
+                                  <option value={station.ID}> {station.ID} </option>
```

Comment:
raczej chcemy podawać `station.name`, jeśli jeszcze nie mamy klasy `Station` na froncie to może tak chwilowo zostać, ale dodaj 
```
// TODO(krolikj): change to station.name once we add station functionality 
```
.